File: 08-DesignPatterns-Java/Solution/SimpleFactory/src/Dices/LoadedDice.java
package Dices;

public class LoadedDice implements IDice {
    private final int _fixedRoll;

    public LoadedDice(int fixedRoll) {
        _fixedRoll = fixedRoll;
    }

    @Override
    public int roll() {
        return _fixedRoll;
    }
}

File: 10-DesignPatterns-CSharp/Solution/SimpleFactory/Dices/LoadedDice.cs
namespace SimpleFactory.Dices
{
    public class LoadedDice : IDice
    {
        private readonly int _fixedRoll;

        public LoadedDice(int fixedRoll) => _fixedRoll = fixedRoll;

        public int Roll() => _fixedRoll;
    }
}
